rllib/core/rl_module/marl_module.py

```diff
@@ -1,6 +1,5 @@
-import abc
```

Comment:
It doesn't make sense to have an inheritance from MultiAgentRLModule to `DefaultMultiAgentRLModule`. This thing is essentially a fancy policy_map that can potentially have neural networks inside for weird multi-agent scenarios.